fairlearn/metrics/_group_metric_set.py

```diff
@@ -93,72 +86,85 @@
 REGRESSION_METRICS[GROUP_ZERO_ONE_LOSS] = group_zero_one_loss
 
 
-def create_group_metric_set(model_type,
-                            y_true,
-                            y_preds,
-                            sensitive_features,
-                            model_titles=None,
-                            sensitive_feature_names=None,
-                            extra_metrics=None):
-    """Create a dictionary matching the Dashboard's cache."""
-    if extra_metrics is not None:
-        raise NotImplementedError("No support for extra_metrics yet")
+def _process_feature_to_integers(feature):
```

Comment:
This reminds me of #339 a lot. I don't know if you'd consider the examples @adrinjalali  posted there as simpler or better, but we can always consolidate such logic later on if need be.

Reply:
Good point about `LabelEncoder` - have revised this to match